google-cloud-clients/google-cloud-contrib/google-cloud-logging-logback/src/test/java/com/google/cloud/logging/logback/MDCEventEnhancerTest.java

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright 2017 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.logging.logback;
+
+import static com.google.common.truth.Truth.assertThat;
+import static org.easymock.EasyMock.anyObject;
+import static org.easymock.EasyMock.capture;
+import static org.easymock.EasyMock.expectLastCall;
+import static org.easymock.EasyMock.replay;
+import static org.easymock.EasyMock.verify;
+
+import ch.qos.logback.classic.Level;
+import ch.qos.logback.classic.spi.LoggingEvent;
+import com.google.cloud.Timestamp;
+import com.google.cloud.logging.LogEntry;
+import com.google.cloud.logging.Logging;
+import com.google.cloud.logging.Logging.WriteOption;
+import com.google.cloud.logging.Severity;
+import java.util.Map;
+import org.easymock.Capture;
+import org.easymock.EasyMock;
+import org.easymock.EasyMockRunner;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.slf4j.MDC;
+
+@RunWith(EasyMockRunner.class)
+public class MDCEventEnhancerTest {
+  private final String projectId = "test-project";
+  private final String filteredMVCEventEnhancerClassName =
+      FilteredMVCEventEnhancer.class.getCanonicalName();
+  private Logging logging;
+  private LoggingAppender loggingAppender;
+
+  class TestLoggingAppender extends LoggingAppender {
+    @Override
+    String getProjectId() {
+      return projectId;
+    }
+
+    @Override
+    Logging getLogging() {
+      return logging;
+    }
+  }
+
+  @Before
+  public void setUp() {
+    logging = EasyMock.createStrictMock(Logging.class);
+    loggingAppender = new TestLoggingAppender();
+  }
+
+  @Test
+  public void testDefaultMDCEventEnhancer() {
```

Comment:
I would probably leave this test in the `AppenderTest` as we're testing the appender's default behavior. If you want to test the MDCEventEnhancer directly, I'd put that test here.

Reply:
That is fair